RELEASE_NOTES.rst

```diff
@@ -1,4 +1,140 @@
-coala 0.7.0 - baloo
+coala 0.8.0 - grizzly
+=====================
+
+::
+
+          _|\____/|_
+         //  \    /  \
+        //   o    o  |\________________
+       /|            |    \            \
+      | |      []    |     |       \    \
+      |  \   \____/ /       \       |    \
+      |   \________/         "      "     |
+      |                                   |
+      |                  \          \     |
+       \       \         |\          |    |
+        \      /\        | \         |    |
+        |     |  \       |  \        |     \
+        |     |   |     /    |      /\      |
+        |    /    |    |     |     /  |      \
+       /_____}   /_____}    /______}  {_______}
+
+^ Fabian, gimme something better than this :P
+                                                                                                 
+
+For this release, we have had 45 developers from around the world contributing
+over 148 commits in the last 8 weeks.
+
+Improving the API available for bear writers is one of the areas we've focused
+on for this release, with several new and exciting features. General performance
+has also been improved heavily with some major changes under the hood. The
+documentation has also been worked on, with an emphasis on user-friendliness
+and design.
+
+There have also been major internal changes in preparation for the complete
+decentralization of bears, which would allow the installation of individual
+bears.
+
+Below are some of the important changes introduced for this release:
+
+**New Features**
+
+- coala now supports syntax highlighting in results!
+
+- Questions are now printed in color; this will improve visibility when a lot
+  of text is written to the screen.
+
+- ``coala-json`` now supports ``--show-bears`` and ``--filter-by-language``
+
+- Added a ``--show-capabilities`` flag that displays the types of issues coala
+  can detect and fix for a particular language.
+
+- Display the line number when a line is missing; this could happen if a bear
+  that had run previously overwrites it.
+
+**For Bear Writers**
```

Comment:
I asked him, he said he still has that unfixed bug and wants to move it for next release.

Forgot about the bear creation tool, sorry! I'll add that in.